src/tests/regressionTests.test.tsx

```diff
@@ -1212,4 +1212,17 @@ describe("regression tests", () => {
     expect(h.elements[0].groupIds).toHaveLength(0);
     expect(h.elements[1].groupIds).toHaveLength(0);
   });
+
+  it("keeps selected element selected when click hit's element bounding box but doesn't hit the element", () => {
+    clickTool("ellipse");
+    mouse.down(0, 0);
+    mouse.up(100, 100);
+
+    clickOnBoundingBoxButNotOnElement();
+    expect(getSelectedElements().length).toBe(1);
+
+    function clickOnBoundingBoxButNotOnElement() {
+      mouse.click(0, 0);
+    }
```

Comment:
Not that I don't like this but it seems inconsistent in style
```suggestion
    // click on bounding box but not on element
    mouse.click(0, 0);
    expect(getSelectedElements().length).toBe(1);
```

Reply:
At the time, I hesitated between commenting and putting it on a function. But style-wise, the comment is more consistent indeed.